Makefile

```diff
@@ -10,10 +10,10 @@ DJANGO_VERSION=django22
 .PHONY: requirements develop clean diff.report view.diff.report quality static
 
 requirements:
-	pip install -q -r requirements/base.txt
+	pip3 install -q -r requirements/base.txt
```

Comment:
if `make upgrade` is run under python 3 virtual environment then we won't need to specify `pip3` explicitly?